### DISCUSSION TITLE
Text and icon with different colors in q-btn

### DISCUSSION THREAD

**Discussion by josejachuf** [quasarframework/quasar#11028] [General - Components / Directives / etc]
Hello there

I am trying, without success, that the text and icon of a q-btn are different. This is using the icon attribute 

What I tried was with STYLE SCOED was to change the q-icon class.
doing the following if I can control it, but I want to avoid it 

```
<q-btn
  label="My Label"
/>
  <q-icon name="zzz" class="myicon"/>
</q-btn
>
```

Grateful if someone helps me conveniently 
Jose

**Comment by MilosPaunovic** [comment]
What you're looking for is the `text-color` prop of `QBtn`.

**Reply by josejachuf** [reply]
Yes, but change both elements, icon and text

**Reply by josejachuf** [reply]
It would be good to have an **icon-color**

**Reply by MilosPaunovic** [reply]
Something like this:
https://codepen.io/milospaunovic/pen/PoKNBXg?editors=101

**Reply by josejachuf** [reply]
Thanks @MilosPaunovic 

Yes, I did that (I put it in the first message), but I wanted to see an alternative to this, I wanted to avoid using it. Because the icon I want it above, I have to change the top and they are all buttons of different size, then they are I have to have a class for each type of button and I want to avoid it. If you do not solve it otherwise, that will be the solution

**Reply by MilosPaunovic** [reply]
You could target icon with CSS, like this:
[https://codepen.io/milospaunovic/pen/PoKNBXg](https://codepen.io/milospaunovic/pen/PoKNBXg)

**Comment by josejachuf** [comment]
```
.q-icon
    color: #D86905!important
```

Now with q-icon it works as I want, but when I indicate in style as scoped stops working

**Reply by MilosPaunovic** [reply]
Check the CSS in the codepen I've sent above.

**Reply by josejachuf** [reply]
Excellent. This worked. What I did, based on your code, was something like a class inheritance and only affects my buttons.

Thank you
